tests/test_hydra.py

```diff
@@ -497,17 +497,13 @@ def test_sweep_complex_defaults(
 --run,-r : Run a job
 --multirun,-m : Run multiple jobs with the configured launcher
 --shell_completion,-sc : Install or Uninstall shell completion:
-    Install:
-    eval "$(python examples/tutorial/2_config_file/my_app.py -sc install=SHELL_NAME)"
+    Bash:
+    eval "$(python examples/tutorial/2_config_file/my_app.py -sc install=bash)"
+    eval "$(python examples/tutorial/2_config_file/my_app.py -sc uninstall=bash)"
```

Comment:
```suggestion
    Bash - Install:
    eval "$(python examples/tutorial/2_config_file/my_app.py -sc install=bash)"
    Bash - Uninstall:
    eval "$(python examples/tutorial/2_config_file/my_app.py -sc uninstall=bash)"
```

Reply:
We can also potentially introduce a shell specific help, but this could be a bit dicey:
```
$ python foo.py --help -sc
Supported shells: bash, fish
For shell specific help, use : --help -sc NAME 

$ python foo.py --help -sc bash
Bash - Install:
  eval "$(python examples/tutorial/2_config_file/my_app.py -sc install=bash)"
Bash - Uninstall:
  eval "$(python examples/tutorial/2_config_file/my_app.py -sc uninstall=bash)"
```

Reply:
That looks quite extensible but I'm not sure if this improves user experiences atm. As a user, I'd prefer using one command (python foo.py --hydra-help) to get everything I need in one page.

Reply:
Okay. at this point I think it's okay but eventually too much of the help screen would be dedicated to this.
I can live what your current proposal, but update it to both the shell and the commands in the header as I suggested.